pkg/interfaces/contracts/vault/ICompositeLiquidityRouter.sol

```diff
@@ -72,35 +76,39 @@ interface ICompositeLiquidityRouter {
      * @notice Remove proportional amounts of underlying from an ERC4626 pool, burning an exact pool token amount.
      * @dev An "ERC4626 pool" contains IERC4626 yield-bearing tokens (e.g., waDAI).
      * @param pool Address of the liquidity pool
+     * @param isWrappedToken An array indicating whether the output token is a wrapper or underlying token
      * @param exactBptAmountIn Exact amount of pool tokens provided
-     * @param minUnderlyingAmountsOut Minimum amounts of underlying tokens out, sorted in token registration order of
+     * @param minAmountsOut Minimum amounts of underlying tokens out, sorted in token registration order
```

Comment:
```suggestion
     * @param minAmountsOut Minimum amounts of underlying tokens out, sorted in token registration order of
```

Reply:
Adding "of" at the end doesn't make sense; did you mean to suggest that?